### DISCUSSION TITLE
Unable to create Stored Procedures.

### DISCUSSION THREAD

**Discussion by MrOkiDoki** [planetscale/discussion#252] [Bug Reports] [answered]
In 'Default' branch, not 'Production' branch, I am unable to create a Stored procedure on a newly created, empty database.

`
CREATE PROCEDURE 'test_procedure' ()
BEGIN
select 1;
END
`

Result:
`Error 1105: syntax error at position 15 near 'procedure' SQL Statement:  DROP procedure IF EXISTS 'test_procedure' `

I am not sure if I am missing something here.

**Comment by jgreet** [comment, marked as answer]
Hi, unfortunately stored procedures (and the other types of stored routines) are not currently available on PlanetScale. We generally recommend moving that sort of logic to the application level.

For more compatibility notes see: https://planetscale.com/docs/reference/mysql-compatibility

Related discussions: https://github.com/planetscale/discussion/discussions/242, https://github.com/planetscale/discussion/discussions/106, https://github.com/planetscale/discussion/discussions/189
